public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -173,7 +173,7 @@ class MetricsPanelCtrl extends PanelCtrl {
 
   updateTimeRange(datasource?: DataSourceApi) {
     this.datasource = datasource || this.datasource;
-    this.range = this.timeSrv.timeRange();
+    this.range = cloneDeep(this.timeSrv.timeRange());
```

Comment:
note, this clone is not needed it was a test to see if there was some mutable thing causing this but it was not